Migrate blog posts to nikola

oraNod opened the pull request:
Relates to #300 

This PR adds a playbook that does a bulk download and conversion of blog posts from the current "ansible.com" to markdown. The resulting markdown isn't perfect and contains a bunch of unnecessary stuff. On the bright side it seems like all the content is there and we'd only need an hour or two of manual intervention/clean up it before importing. We could refine the playbook to strip out some of the unnecessary stuff but we'll need to do that as a follow up.

Debatable whether this playbook needs to live in the repo or not.

samccann commented:
Sorry for the delay.

I tried this on two blog posts. The first one (https://www.ansible.com/blog/ansible-data-manipulation-with-a-filter) worked well. There's html cruft at the top and bottom but easy enough to delete.

The second blog post wasn't so good - https://www.ansible.com/blog/event-driven-ansible-is-here

It ended up with a lot of html-like cruft interspersed in the file.

```
[As you may recall, we introduced Event-Driven Ansible in developer
preview last fall at AnsibleFest. Since that time, much work has been
done across the community, the Red Hat development teams, customers, and
last but not least, Red Hat partners. Today,
]{style="color: #000000;"}[we are pleased to
announce](https://www.redhat.com/en/about/press-releases/red-hat-accelerates-it-automation-event-driven-ansible)[
that Event-Driven Ansible will be concluding its developer preview and
will become generally available as part of Red Hat Ansible Automation
Platform 2.4.  ]{style="color: #000000;"}

[If you are new to Event-Driven Ansible, check out
the]{style="color: #000000;"}[ developer preview
blog](https://www.ansible.com/blog/introducing-event-driven-ansible)[ I
wrote last fall to learn the basics, and you may also be interested in
this ]{style="color: #000000;"}[video on Ansible
Rulebooks](https://www.youtube.com/watch?v=PtevBKX1SYI&list=PLdu06OJoEf2a3fFl6uaoyGV526ilwD97R&index=3)[,
as well as others in this playlist. ]{style="color: #000000;"}
```

samccann commented:
Sorry for the delay. I tested this out on a couple of blog posts:
https://www.ansible.com/blog/ansible-data-manipulation-with-a-filter  -worked fine. Had to delete html cruft at the top and bottom but otherwise a quick scan suggests it worked well.

https://www.ansible.com/blog/event-driven-ansible-is-here - this left a scattering of html in the middle of the file as shown below. I suppose we could manually delete it but wondering if something can be 'tweaked' somewhere to clean it up?

```
[As you may recall, we introduced Event-Driven Ansible in developer
preview last fall at AnsibleFest. Since that time, much work has been
done across the community, the Red Hat development teams, customers, and
last but not least, Red Hat partners. Today,
]{style="color: #000000;"}[we are pleased to
announce](https://www.redhat.com/en/about/press-releases/red-hat-accelerates-it-automation-event-driven-ansible)[
that Event-Driven Ansible will be concluding its developer preview and
will become generally available as part of Red Hat Ansible Automation
Platform 2.4.  ]{style="color: #000000;"}

[If you are new to Event-Driven Ansible, check out
the]{style="color: #000000;"}[ developer preview
blog](https://www.ansible.com/blog/introducing-event-driven-ansible)[ I
wrote last fall to learn the basics, and you may also be interested in
this ]{style="color: #000000;"}[video on Ansible
Rulebooks](https://www.youtube.com/watch?v=PtevBKX1SYI&list=PLdu06OJoEf2a3fFl6uaoyGV526ilwD97R&index=3)[,
as well as others in this playlist. ]{style="color: #000000;"}
```

oraNod commented:
@samccann Thanks for trying it out! I've cleaned up the results with a bunch of sed expressions that remove all those HTML blocks before the first level heading and at the bottom.

I don't think we're going to need to commit the playbook to the repository though. I'll update this PR later today or tomorrow with the batch of migrated blog posts. I'd like to keep it in the same PR so there's a bit more history behind the full effort to migrate the posts.

When I do update the PR I was hoping maybe we could ping @Dule-martins and his team to take a look and possibly do a clean up effort or something like that. It'd be good to get more eyes on the migrated content and double check the results.

oraNod commented:
@samccann OK, I've pushed a commit that adds all the posts we need to migrate over, as mentioned in #300 

Here are the steps I took:

- Ran the playbook to pull down all the HTML content and convert it to MD files.
- Removed - with sed expressions - all the extraneous HTML bits from each document.
- Put everything into a single `posts/migrated` folder
- Cleaned up all the nikola metadata and converted the dates to match the format nikola expected.
- Made sure all the headings are correctly formatted and in the right order
- Fixed a few issues with code blocks
- Resolved some minor formatting issues that were breaking the nikola builds

It looks like the migrated blog posts are rendered now and the content is migrated over. Some issues still persist though.

Next steps will be to:

- Pull down all the required images. For convenience as well as accessibility purposes, I think we should remove any images that are "decorative". There are a few posts where images really accompany the text so we'll need to go through and save them all to an `images` folder and then update the markdown references.
- Go through the markdown files and clean up some of the random HTML cruff that was the result of the conversion process.
- Go through the markdown files and clean up some formatting. For example, make sure anchors are on the same line so they render correctly as a link.

I think we can handle these next steps as a series of follow up PRs.

samccann reviewed (dismissed):
Okay I can see the blog posts rendered now in the CI-generated website so marking approved.

samccann commented:
So given we will merge this Pr with all the converted blog posts, are we then asking @Dule-martins to lead the effort with the community-writers to remove the extra HTML and port over any missing images?

oraNod commented:
> So given we will merge this Pr with all the converted blog posts, are we then asking @Dule-martins to lead the effort with the community-writers to remove the extra HTML and port over any missing images?

I would like to do that, yes. It seems like it would be a good follow up to their previous efforts. I'll create separate issues with details and task lists so the request is clear and focused on specific outcomes.

oraNod commented:
@samccann FYI I would like to get approvals from @GregSutcliffe and @leogallego before we merge this. They might have a few nits.

One thing to consider is the name of the markdown folder under the `posts` directory. This is used as a url context that the published posts will be under when we go live.

`ansible.com/blog/migrated/<post_name>`

Should we change "migrated" to "archive"? Is there a more suitable option?

oraNod commented:
@samccann Here is one of the follow up issues for dealing with image references: https://github.com/ansible-community/community-website/issues/353

samccann commented:
Thanks for those issues! Very detailed and should be easy enough for the community writers to take over.

As for the directory, yeah maybe archive is better.

oraNod commented:
@samccann I renamed the folder from "migrated" to "achive". I think that's a better fit and makes the most sense for users (should anyone happen to even notice that in the url on the live site). Had to force push though, which reset your previous approval...

GregSutcliffe commented:
I'll add an explicit approval later when I've had a little more time with it, but in short:
- conversion looks pretty good in a lot of cases, the code blocks and tables especially
- agree with cleaning up some of the older posts in follow-up PRs
- agree with `archive` as the holding folder 

Do we need to ensure the original URLs still work? Nikola can create redirects for them if needed.

oraNod commented:
That's a good idea about the redirects @GregSutcliffe we should probably set those up.

After migration the url will change. `https://www.ansible.com/blog/connecting-to-a-windows-host` > `https://www.ansible.com/blog/archive/connecting-to-a-windows-host`

oraNod commented:
Although we should probably do the redirects in a follow on PR

oraNod commented:
An alternative to adding a bunch of redirects might be to move all the markdown files in the `archive` holding folder up a level to the root of the `posts` directory. That avoids the `archive` context in the url and the post names should all stay the same under "ansible.com".

samccann commented:
/archive/ was helpful in denoting 'a bunch of old stuffs' but if we can get away w/o redirects by putting it all in the posts/ dir, then I say go for it.

GregSutcliffe commented:
I think I've been unclear. I'm not talking about where the files live, that's irrelevant - I'm talking about the URL of the post. *By default*, yes, the path becomes part of the URL, hence we see "archive" in these posts and datestamps in the existing posts, that's coming from the file structure. However, we can override it if we wish.

(I haven't got this PR checked out, so I'm going to demo on Hardy's Galaxy post, but the concepts are the same)

In most SSG engines you can add URL aliases to the YAML frontmatter, but Nikola can't do that. You *can* add redirects to the `conf.py` file, like this:
`REDIRECTIONS = [("/blog/welcome-to-the-ansible-lightspeed-technical-preview.html", "/blog/archive/welcome-to-the-ansible-lightspeed-technical-preview/")]`
In theory this then means that *both* URLs work, with the former being a redirect to the latter, but in my testing it didn't work especially well (although this might be an issue with the built in web engine, Apache/Nginx might work fine - I can go into details). 

Another alternative is to put them somewhere else entirely. By moving `/posts/archive` to `/archive` and then adding `("archive/*.md", "blog", "post.tmpl")` to the POSTS directive in `conf.py`, we can remove the `archive` part of the URL and everything seems to work fine. That might be the better option? Thoughts @oraNod?

oraNod commented:
@GregSutcliffe Thanks for looking at this. The option about the posts directive in `conf.py` sounds promising. In any case I would prefer to avoid redirects or another web server config. IMO that seems like it would be a pain in the event we need to migrate hosts at some point in future.

GregSutcliffe commented:
I just retested the redirects and my concerns seem to have gone - after a complete clean (rm cache/output) and rebuild it appears to work.

So, I think I lean towards a list of redirects for these old posts - it should be easy to generate and we can do it in a followup PR. That is then contained within Nikola and doesn't require any special config at the webserver level, and thus wouldn't affect any host migrations either.

oraNod commented:
@GregSutcliffe Awesome. I just retested what I mentioned in the previous comment and that doesn't work. I'm not honestly sure why I thought moving the markdown files to folders would result in matching urls.

I agree that nikola redirects are the way to go. In fact I found out how to do this with mk-docs yesterday so kind of cool timing.

Anyway thanks again for the input on this PR. That level of scrutiny was needed. I can create a follow up issue for the redirect tuples.

oraNod commented:
@GregSutcliffe and @leogallego Hey, final nag for feedback. I'll merge this one later today or tomorrow. Thanks!

leogallego commented on posts/archive/an-introduction-to-windows-security-with-ansible.md:
```diff
@@ -0,0 +1,167 @@
+
+###  Confidentiality
+
+[As Bianca mentioned in the [first
```
Extra `[`

leogallego commented on posts/archive/ansible-and-infoblox-roles-deep-dive.md:
```diff
@@ -0,0 +1,354 @@
+
+### Infoblox credentials and the nios_provider
+
+[Any time you use Ansible with Infoblox, invoking an Infoblox
```
extra [

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M Blog
+1](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20Blog%201.png?width=1600&name=Colin%20M%20Blog%201.png){width="1600"
+style="width: 1600px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20Blog%201.png?width=800&name=Colin%20M%20Blog%201.png 800w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20Blog%201.png?width=1600&name=Colin%20M%20Blog%201.png 1600w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20Blog%201.png?width=2400&name=Colin%20M%20Blog%201.png 2400w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20Blog%201.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+2](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%202.png?width=1024&name=Colin%20M%20blog%202.png){width="1024"
+style="width: 1024px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%202.png?width=512&name=Colin%20M%20blog%202.png 512w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%202.png?width=1024&name=Colin%20M%20blog%202.png 1024w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%202.png?width=1536&name=Colin%20M%20blog%202.png 1536w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%202.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+3](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%203.png?width=1600&name=Colin%20M%20blog%203.png){width="1600"
+style="width: 1600px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%203.png?width=800&name=Colin%20M%20blog%203.png 800w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%203.png?width=1600&name=Colin%20M%20blog%203.png 1600w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%203.png?width=2400&name=Colin%20M%20blog%203.png 2400w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%203.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+4](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%204.png?width=1600&name=Colin%20M%20blog%204.png){width="1600"
+style="width: 1600px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%204.png?width=800&name=Colin%20M%20blog%204.png 800w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%204.png?width=1600&name=Colin%20M%20blog%204.png 1600w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%204.png?width=2400&name=Colin%20M%20blog%204.png 2400w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%204.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+6](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%206.png?width=1444&name=Colin%20M%20blog%206.png){width="1444"
+style="width: 1444px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%206.png?width=722&name=Colin%20M%20blog%206.png 722w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%206.png?width=1444&name=Colin%20M%20blog%206.png 1444w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%206.png?width=2166&name=Colin%20M%20blog%206.png 2166w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%206.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+5](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%205.png?width=1106&name=Colin%20M%20blog%205.png){width="1106"
+style="width: 1106px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%205.png?width=553&name=Colin%20M%20blog%205.png 553w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%205.png?width=1106&name=Colin%20M%20blog%205.png 1106w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%205.png?width=1659&name=Colin%20M%20blog%205.png 1659w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%205.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+7](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%207.png?width=1600&name=Colin%20M%20blog%207.png){width="1600"
+style="width: 1600px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%207.png?width=800&name=Colin%20M%20blog%207.png 800w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%207.png?width=1600&name=Colin%20M%20blog%207.png 1600w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%207.png?width=2400&name=Colin%20M%20blog%207.png 2400w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%207.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-linting-with-github-actions.md:
```diff
@@ -0,0 +1,180 @@
+![Colin M blog
+8](https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%208.png?width=1600&name=Colin%20M%20blog%208.png){width="1600"
+style="width: 1600px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%208.png?width=800&name=Colin%20M%20blog%208.png 800w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%208.png?width=1600&name=Colin%20M%20blog%208.png 1600w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%208.png?width=2400&name=Colin%20M%20blog%208.png 2400w, https://www.ansible.com/hs-fs/hubfs/Colin%20M%20blog%208.png?width ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-network-resource-purge-parameter.md:
```diff
@@ -0,0 +1,188 @@
+![network purge blog
+](https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%20.png?width=1203&name=network%20purge%20blog%20.png){width="1203"
+loading="lazy" style="width: 1203px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%20.png?width=602&name=network%20purge%20blog%20.png 602w, https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%20.png?width=1203&name=network%20purge%20blog%20.png 1203w, https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%20.png?width=1805&name=network%20purge%20blog%20.png 1805w, https://www.ansible.com/hs-fs/hubfs/ ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-network-resource-purge-parameter.md:
```diff
@@ -0,0 +1,188 @@
+![network purge blog
+2](https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%202.png?width=837&name=network%20purge%20blog%202.png){width="837"
+loading="lazy" style="width: 837px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%202.png?width=419&name=network%20purge%20blog%202.png 419w, https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%202.png?width=837&name=network%20purge%20blog%202.png 837w, https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%202.png?width=1256&name=network%20purge%20blog%202.png 1256w, https://www.ansible.com/hs-fs/hu ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-network-resource-purge-parameter.md:
```diff
@@ -0,0 +1,188 @@
+![network purge blog
+3](https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%203.png?width=1600&name=network%20purge%20blog%203.png){width="1600"
+loading="lazy" style="width: 1600px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%203.png?width=800&name=network%20purge%20blog%203.png 800w, https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%203.png?width=1600&name=network%20purge%20blog%203.png 1600w, https://www.ansible.com/hs-fs/hubfs/network%20purge%20blog%203.png?width=2400&name=network%20purge%20blog%203.png 2400w, https://www.ansible.com/hs-fs/ ...
```
hub spot specific links, display broken

leogallego commented on posts/archive/ansible-servicenow-howto-part-3-making-outbound-restful-api-calls-to-ansible-tower.md:
```diff
@@ -0,0 +1,350 @@
+
+![image3-4](https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=1913&name=image3-4.png){style="width: 1913px;"
+width="1913"
+srcset="https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=957&name=image3-4.png 957w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=1913&name=image3-4.png 1913w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=2870&name=image3-4.png 2870w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?width=3826&name=image3-4.png 3826w, https://www.ansible.com/hs-fs/hubfs/image3-4.png?widt ...
```
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.

leogallego commented on posts/archive/ansible-servicenow-opening-and-closing-tickets.md:
```diff
@@ -0,0 +1,218 @@
+
+![blog_closing-servicenow-ticket](https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=914&name=blog_closing-servicenow-ticket.png){style="width: 914px;"
+width="914"
+srcset="https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=457&name=blog_closing-servicenow-ticket.png 457w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=914&name=blog_closing-servicenow-ticket.png 914w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=137 ...
```
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.

leogallego commented on posts/archive/ansible-tips-and-tricks-dealing-with-unreliable-connections-and-services.md:
```diff
@@ -0,0 +1,260 @@
+![Terminal
+Readout](https://www.ansible.com/hs-fs/hubfs/Images/blog-social/Ansible-Networking-Tips-Tricks-Terminal-Screenshot.png?width=884&name=Ansible-Networking-Tips-Tricks-Terminal-Screenshot.png){style="width: 884px;"
+width="884"
+srcset="https://www.ansible.com/hs-fs/hubfs/Images/blog-social/Ansible-Networking-Tips-Tricks-Terminal-Screenshot.png?width=442&name=Ansible-Networking-Tips-Tricks-Terminal-Screenshot.png 442w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/Ansible-Networking-Tips-Tricks-Terminal-Screenshot.png?width=884&name=Ansible-Networking-Tips-Tricks-Terminal-Screenshot.png 884w, https://www.ansible ...
```
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.

leogallego commented on posts/archive/taking-automation-to-the-next-level-using-ansible-gitops-to-manage-the-lifecycle-of-a-containerized-application.md:
```diff
@@ -0,0 +1,497 @@
+[![taking automation to the next level
+blog](https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=1200&name=taking%20automation%20to%20the%20next%20level%20blog.png){width="1200"
+loading="lazy" style="width: 1200px;"
+srcset="https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=600&name=taking%20automation%20to%20the%20next%20level%20blog.png 600w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=1200&name=taking%20automation%20to%20the%20next%20level%20blog.png 1200w, https://www.ansible.com/hs-fs/hubfs/taking%20au ...
```
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.

leogallego commented on posts/archive/taking-automation-to-the-next-level-using-ansible-gitops-to-manage-the-lifecycle-of-a-containerized-application.md:
```diff
@@ -0,0 +1,497 @@
+srcset="https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=600&name=taking%20automation%20to%20the%20next%20level%20blog.png 600w, https://www.ansible.com/hs-fs/hubfs/taking%20automation%20to%20the%20next%20level%20blog.png?width=1200&name=taking%20automation%20to%20the%20next%20level%20blog.png 1200w, https://www.ansible.com/hs-fs/hubfs/taking%20au ...
+sizes="(max-width: 1200px) 100vw, 1200px"}]{style="color: #000000;"}
+
+[One of the great advantages of combining GitOps with Ansible is that
```
false link

leogallego commented on posts/archive/taking-automation-to-the-next-level-using-ansible-gitops-to-manage-the-lifecycle-of-a-containerized-application.md:
This one has a lot of formatting issues for some reason, plenty of false links `[ ]`.

leogallego commented on posts/archive/whats-new-cloud-automation-with-amazon.aws-6.0.0.md:
```diff
@@ -0,0 +1,363 @@
+
+The following features have been removed from this collection release.
+
++-----------------------------------+-----------------------------------+
```
Broken table

leogallego commented on posts/archive/whats-new-cloud-automation-with-amazon.aws-6.0.0.md:
Formatting issues

leogallego reviewed (approved):
There are quite a few display issues with some resources that have the hubspot URLs for static content like images, some table and formatting errors as well, but all in all the content seems to be good and the clean up can be done in a 2nd round!


Marked a few of the issues I came across for reference for the future review after merge.

oraNod commented:
Thanks for taking such a detailed look @leogallego Yeah there certainly are some issues with formatting and images but I'll tackle them in follow up PRs.
